.travis.yml

```diff
@@ -1,14 +1,13 @@
+sudo: false
+cache: bundler
 language: ruby
 
 rvm:
-  - 2.0
```

Comment:
Please be focused on your pull request.  You shouldn't need to drop support for ruby 2.0 for code style fixes.